ntrans_dataprep.py

```diff
@@ -91,9 +107,24 @@ def generate_ngrams_from_corpus():
         root="BNC/Texts/", fileids=r"[A-K]/\w*/\w*\.xml"
     )
 
+    # Tells the program how to name the generated csv-files when splitting data into several
+    # files.
+    data_chunk = 1
```

Comment:
If you need a comment like this, then the variable name isn't good enough. How about `chunk_number`?